mllib/src/main/scala/org/apache/spark/ml/optim/WeightedLeastSquares.scala

```diff
@@ -200,7 +200,7 @@ private[ml] object WeightedLeastSquares {
      * Adds an instance.
      */
     def add(instance: Instance): this.type = {
-      val Instance(l, w, f) = instance
+      val Instance(label, w, f) = instance
```

Comment:
Feel free to just write out weight and feature too, while we're touching this, for consistency